lib/screens/type_of_glass/type_of_glass.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class TypeOfGlassPage extends StatelessWidget {
+  const TypeOfGlassPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Type of glass'),
+      ),
+      body: const Center(
+        child: Text('Type of glass'),
```

Comment:
- [ ] Let's add this raw Strings in the Internationalization standard for future languages implementations